alphadia/workflow/managers/base.py

```diff
@@ -85,12 +85,12 @@ def load(self):
         """Load the state from pickle file."""
         if self.path is None:
             self.reporter.log_string(
-                f"{self.__class__.__name__} not loaded from disk.",
+                f"{self.__class__.__name__}: loading saved state not requested, will be initialized.",
```

Comment:
kind of a weird way from me to notifiy that something unexpected did not happen :D